routemaster/db/tests/test_model_repr.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from routemaster.db import Label, History
+
+
+INSTANCES = [
+    (
+        Label(state_machine='foo', name='bar'),
+        "Label(state_machine='foo', name='bar')",
+    ),
+    (
+        History(
+            label_state_machine='foo',
+            label_name='bar',
+        ),
+        "History(id=None, label_state_machine='foo', label_name='bar')",
+    ),
+]
+
+
+@pytest.mark.parametrize("instance, representation", INSTANCES)
```

Comment:
Single quotes

Reply:
This is done in double quotes elsewhere in the codebase, will move them to single quotes in a separate branch.